pkg/telemetry/cassandra_agent/cassandra_agent_config.go

```diff
@@ -0,0 +1,155 @@
+package cassandra_agent
+
+import (
+	"context"
+	"path/filepath"
+	"time"
+
+	cassdcapi "github.com/k8ssandra/cass-operator/apis/cassandra/v1beta1"
+	k8ssandraapi "github.com/k8ssandra/k8ssandra-operator/apis/k8ssandra/v1alpha1"
+	telemetryapi "github.com/k8ssandra/k8ssandra-operator/apis/telemetry/v1alpha1"
+	"github.com/k8ssandra/k8ssandra-operator/pkg/annotations"
+	"github.com/k8ssandra/k8ssandra-operator/pkg/cassandra"
+	"github.com/k8ssandra/k8ssandra-operator/pkg/labels"
+	"github.com/k8ssandra/k8ssandra-operator/pkg/result"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/yaml"
+)
+
+var (
+	agentConfigLocation = "/opt/management-api/metric-collector.yaml"
```

Comment:
```suggestion
	agentConfigLocation = "/opt/management-api/configs/metric-collector.yaml"
```

As you can see [here](https://github.com/k8ssandra/management-api-for-apache-cassandra/commit/f631db71b4f4bfefb3791e42b080e6c2df3f7fc5#diff-813a5eb5912d07588a0a72cf84860f552a482d75d77b91546d4125e53df29258R11) the right location is under `$MAAC_PATH/configs/...`.
I've successfully tested a custom filter after switching to that path. Otherwise my config was ignored.

FTR, another way of making it work with the current path is to set the [CONFIG_PATH_PROPERTY](https://github.com/k8ssandra/management-api-for-apache-cassandra/commit/f631db71b4f4bfefb3791e42b080e6c2df3f7fc5#diff-813a5eb5912d07588a0a72cf84860f552a482d75d77b91546d4125e53df29258R15) accordingly, but I think the right default path is all we need.

Reply:
Good catch, thanks. I've pushed another commit. I wasn't sure if it was right (hence why I [asked](https://github.com/k8ssandra/k8ssandra-operator/pull/813#issuecomment-1406123112)) but I obviously misread [this](https://github.com/k8ssandra/k8ssandra-operator/issues/815#issuecomment-1396508708) answer quite badly - sorry!